.github/workflows/issue_notifications.yml

```
@@ -0,0 +1,53 @@
+      - name: Send notification to Discord
+        env:
+          DISCORD_WEBHOOK_URL: ${{ secrets.DISCORD_WEBHOOK_URL }}
+          ACTION: ${{ github.event.action }}
+          NUMBER: ${{ github.event.issue.number }}
+          TITLE: ${{ github.event.issue.title }}
+          URL: ${{ github.event.issue.html_url }}
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const axios = require('axios');
+            
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)